src/com/aca/narekhakobyan/homework/items/models/PenColor.java

```diff
@@ -0,0 +1,6 @@
+package com.aca.narekhakobyan.homework.items.models;
+
+public enum PenColor {
```

Comment:
Move it into PenAudioRecorder

Reply:
@garikKalash If I put it into PenAudioRecorder then how I can initialize list of gadgets from main if it contains PenAudioRecorder?

Reply:
@garikKalash I mean how should I pass, for example, PenColor.Blue parameter?